ports/paho-mqtt/portfile.cmake

```
@@ -1,37 +1,44 @@
 vcpkg_from_github(
   OUT_SOURCE_PATH SOURCE_PATH
   REPO eclipse/paho.mqtt.c
-  REF 3b7ae6348bc917d42c04efa962e4868c09bbde9f # v1.3.9
-  SHA512 73c10b7da7aa228100511db280ae56484cb8c42b8f0cfafb2fa3f6e230b4bb1d6b3611aa9219736a0baa9d7de0baf802dd70dbf308077f1a745bd61a67a797c7
+  REF eff2da4875725509fc97641b118d6badf3e3a08f # v1.3.10
+  SHA512 5e190ffd978dad0f60b70bdad2727e567539aff2c55e38023dfbcbe9b50bcbe75d8230c501c09e9c0a0329bed3fd149d5d59d65ad0ba0f84a54f641b4f01390a
   HEAD_REF master
   PATCHES
-    remove_compiler_options.patch
-    fix-install-path.patch
     fix-unresolvedsymbol-arm.patch
 )
 
 string(COMPARE EQUAL "${VCPKG_LIBRARY_LINKAGE}" "static" PAHO_BUILD_STATIC)
 string(COMPARE EQUAL "${VCPKG_LIBRARY_LINKAGE}" "dynamic" PAHO_BUILD_DYNAMIC)
 
-vcpkg_configure_cmake(
-  SOURCE_PATH ${SOURCE_PATH}
-  PREFER_NINJA
+vcpkg_cmake_configure(
+  SOURCE_PATH "${SOURCE_PATH}"
   OPTIONS
     -DPAHO_WITH_SSL=TRUE
     -DPAHO_BUILD_SHARED=${PAHO_BUILD_DYNAMIC}
     -DPAHO_BUILD_STATIC=${PAHO_BUILD_STATIC}
     -DPAHO_ENABLE_TESTING=FALSE
 )
 
-vcpkg_install_cmake()
-vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/eclipse-paho-mqtt-c TARGET_PATH share/eclipse-paho-mqtt-c)
+vcpkg_cmake_install()
+vcpkg_cmake_config_fixup(PACKAGE_NAME eclipse-paho-mqtt-c CONFIG_PATH lib/cmake/eclipse-paho-mqtt-c)
 vcpkg_copy_pdbs()
 
 if(VCPKG_LIBRARY_LINKAGE STREQUAL "dynamic")
     vcpkg_copy_tools(TOOL_NAMES MQTTVersion AUTO_CLEAN)
 endif()
 
+vcpkg_replace_string("${CURRENT_PACKAGES_DIR}/share/eclipse-paho-mqtt-c/eclipse-paho-mqtt-cConfig.cmake"
+    "# Generated by CMake"
+    "# Generated by CMake
+include(CMakeFindDependencyMacro)
+find_dependency(OpenSSL)"
+)
+
+
 file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include" "${CURRENT_PACKAGES_DIR}/debug/share")
+file(COPY "${CURRENT_PACKAGES_DIR}/share/doc" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}")
+file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/share/doc")
 
-file(COPY ${SOURCE_PATH}/README.md DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT})
-file(INSTALL ${SOURCE_PATH}/about.html DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)
+file(COPY "${SOURCE_PATH}/README.md" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}")
+file(INSTALL "${SOURCE_PATH}/about.html" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}" RENAME copyright)
```

```suggestion
file(INSTALL "${SOURCE_PATH}/LICENSE" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}" RENAME copyright)
```

Why are you copying readme?
There is now a `LICENSE` file

---

Thanks for your suggestion, I missed this. Done.